config/crd/patches/webhook_in_freeipas.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: idms.idmocp.redhat.com
+  name: freeipas.idmocp.redhat.com
```

Comment:
This should be  `idms.idmocp.redhat.com` instead of `freeipas.idmocp.redhat.com`.

Now it is fixed.